src/ott/geometry/costs.py

```diff
@@ -496,13 +498,14 @@ class Bures(CostFn):
 
   Args:
     dimension: Dimensionality of the data.
-    kwargs: Keyword arguments for :func:`~ott.math.matrix_square_root.sqrtm`.
+    sqrtm_kw: Dictionary of keyword arguments to control the
+      behavior of inner calls to :func:`~ott.math.matrix_square_root.sqrtm`.
   """
 
-  def __init__(self, dimension: int, **kwargs: Any):
+  def __init__(self, dimension: int, sqrtm_kw: Dict[str, Any] = None):
     super().__init__()
     self._dimension = dimension
-    self._sqrtm_kw = kwargs
+    self._sqrtm_kw = sqrtm_kw if sqrtm_kw is not None else {}
```

Comment:
Slightly shorter: {} if sqrtm_kw is None else sqrtm_kw

Reply:
True! thanks